assets/src/blocks/ShareButtons/ShareButtonsEditor.js

```diff
@@ -1,192 +1,192 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
For me this doesn't remove the linting error, `/* eslint-disable jsx-a11y/label-has-for */` does 🤔

Reply:
I can add the two for that line.